airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -458,3 +458,9 @@
   dockerRepository: airbyte/source-amazon-ads
   dockerImageTag: 0.1.0
   documentationUrl: https://docs.airbyte.io/integrations/sources/amazon-ads
+- sourceDefinitionId: b2e713cd-cc36-4c0a-b5bd-b47cb8a0561e
```

Comment:
lets remove the old mongo db implementation, having two implementations is confusing to the user

Reply:
I removed old implementation from source definitions.